index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="css/style.css" rel="stylesheet">
+  </head>
+
+<body>
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+      <p> The man who saved a billion lives</p>
+  <div id="img-div">
+    <img id ="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+    <div id="img-caption">
+    Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase 
+    wheat yields - part of his life-long war on hunger.
+    </div>
+  </div>
+ <section id="tribute-info">
+    <h3 id="headline"> Here's a time line of Dr.Borlaug's life:</h3>
+      <ul>
+        <li> <strong>1914</strong> - Born in Cresco, Iowa</li>
+        <li> <strong>1933</strong>  - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program 
```

Comment:
- [Optional] Great job for adding the comment politely and with reasons 👍,  I think it would be better if you use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.